project/Component/NefCommon/Models/RenderingURL.swift

```diff
@@ -19,6 +19,12 @@ public struct RenderingURL {
     }
 }
 
+extension Unicode.Scalar {
+    var isAlphanumeric: Bool {
+        CharacterSet.alphanumerics.inverted.contains(self)
+    }
```

Comment:
This is checking exactly the opposite, as you are inverting the character set.

Reply:
puf! you are right, good catch!